doc/releases/migration-guide-3.7.rst

```diff
@@ -92,6 +92,12 @@ MbedTLS
   of other Kconfig options which are now named `CONFIG_MBEDTLS_SHA*`.
 * The `CONFIG_MBEDTLS_MAC_ALL_ENABLED` Kconfig option has been removed. Its equivalent is the
   combination of :kconfig:option:`CONFIG_MBEDTLS_HASH_ALL_ENABLED` and :kconfig:option:`CONFIG_MBEDTLS_CMAC`.
+* When there is any PSA crypto provider available in the system
+  (i.e. :kconfig:option:`CONFIG_MBEDTLS_PSA_CRYPTO_CLIENT` is set), desired PSA crypto
+  features must be explicitly enabled using proper `CONFIG_PSA_WANT_*`. (:github:`72243`)
```

Comment:
```suggestion
  features must be explicitly enabled using proper ``CONFIG_PSA_WANT_*``. (:github:`72243`)
```

Reply:
Just realized that this change is to be done in other entries (mine) that are already merged. Can you do that for me (Mbed TLS entries in the migration guide)? :smiley:

Reply:
Done ;)